crates/metastore/src/database.rs

```diff
@@ -287,6 +287,18 @@ enum CreatePolicy {
     Create,
 }
 
+impl CreatePolicy {
+    pub fn new(if_not_exists: bool, or_replace: bool) -> Self {
+        if or_replace {
+            CreatePolicy::CreateOrReplace
+        } else if if_not_exists {
+            CreatePolicy::CreateIfNotExists
+        } else {
+            CreatePolicy::Create
+        }
+    }
+}
```

Comment:
Could we return an error if both are set to true?

Reply:
why do we need to pass two bools in the first place? (which presumably someone can get wrong still?) 

my gut wants it to be possible to return a `CreatePolicy` as a method on the `Mutation::CreateTable` to avoid the awkward and error prone calling problem, but...

Reply:
So it stems from how the sqlparser interprets the string. 

`CREATE OR REPLACE TABLE IF NOT EXISTS` gets parsed to 
```rust
CreateTable {or_replace: true, if_not_exists: true}
```

We could try to handle this closer to the parser, or even in the parser itself.

Reply:
If that's directed at my comment, I was think we would just return an error from `new` since it's closest to where this is considered "unsupported". We may support it in the future, idk, but I just want to make sure we're not putting something in a potentially unknown state by not explicit what is supported and what isn't.